### PR TITLE
Feed solvent wellrates to data::wells for summary output

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

File: opm/autodiff/WellStateFullyImplicitBlackoilSolvent.hpp [hunk 1]
```diff
@@ -33,6 +33,37 @@ namespace Opm
...
+
+            const int nw = WellState::numWells();
+
+            // If there are now wells numPhases throws a floating point
+            // exception.
+            if (nw == 0) {
+                return res;
+            }
+
+            const int np = BaseType::numPhases();
```
Comment (jokva) [inline opm/autodiff/WellStateFullyImplicitBlackoilSolvent.hpp]: This will break when there are no wells (division-by-zero).

Comment (atgeirr) [inline opm/autodiff/WellStateFullyImplicitBlackoilSolvent.hpp]: Take a look a few lines above :-)

**Comment by atgeirr** [comment]
Looks to me like my two minor issues have been fixed, will merge when green!

**Comment by atgeirr** [comment]
Since the cause of Travis failure is in opm-upscaling and unrelated to this, I'll merge.

**Pull request merged**